### ISSUE TITLE
FAQ: Analytics

### ISSUE THREAD

**Issue by hbillings** [NUKnightLab/TimelineJS3#233]
A question came through NICAR about whether it's possible to have analytics on a Timeline. Seems like something worth an FAQ note.

**Comment by JoeGermuska** [comment]
Agreed. While it hasn't been widely tested, as designed people should be able to configure the `ga_property_id` option documented at http://timeline3.knightlab.com/docs/options.html

Might be nice to work out a test, and to consider exactly how we want to support it for CDN embeds (as it is now, it would probably clobber our own analytics, but we might want to have two trackers instead, since it's important to us to know how embeds are used.)
